### PR TITLE
develop

### PR DESCRIPTION
**Pull request by Ch-Tima**
Added company public page
- components: 'company-page' and 'image-slider'
- 'FilterPipe.ts' to search for services on the "company page"
 - change in database:
   * CompanyImage
   * CompanyOpenHour

### CODE REVIEW

**Comment by Ch-Tima** [comment]
In "AdminPanel" I added a panel to manage "location". From #10

File: DAL/Repository/CompanyImagesRepository.cs
Comment (AN0NCER) [inline DAL/Repository/CompanyImagesRepository.cs:1]:
```c#
using DAL.Context;
using Domain.Models;

namespace DAL.Repository
{
    /// <summary>
    /// Repository for working with company images.
    /// </summary>
    public class CompanyImagesRepository : BaseRepository<CompanyImage, string>
    {
        /// <summary>
        /// Company Image Repository Builder.
        /// </summary>
        /// <param name="db">Database context.</param>
        public CompanyImagesRepository(BeautyBookDbContext db) : base(db)
        {
        }
    }
}
```

File: DAL/Repository/CompanyOpenHoursRepository.cs
Comment (AN0NCER) [inline DAL/Repository/CompanyOpenHoursRepository.cs:1]:
```c#
using DAL.Context;
using Domain.Models;

namespace DAL.Repository
{
    /// <summary>
    /// Repository for working with business hours of companies.
    /// </summary>
    public class CompanyOpenHoursRepository : BaseRepository<CompanyOpenHours, string>
    {
        /// <summary>
        /// Company Hours Repository Builder.
        /// </summary>
        /// <param name="db">Database context.</param>
        public CompanyOpenHoursRepository(BeautyBookDbContext db) : base(db)
        {
        }
    }
}
```

**Comment by Ch-Tima** [comment]
Thank you. 
I'll change it.

**Comment by Ch-Tima** [comment]
I've added comments to "back-end" #15

**Pull request merged**